tests/test_connectables.py

```diff
@@ -110,3 +114,12 @@ def test_terse_implicit_image_no_repository():
     id = "bar"
     validator.validate(id)
     assert Connectable.from_str(id).to_terse() == "bar"
+
+
+def test_container_connectable():
+    id = "container:bar"
```

Comment:
Use some realistic value here please -- (i.e. at least make it be some hex string).

You also should modify the JSON Schema for `connectables` to validate that kind of thing, and have a test which triggers that validation (see what the existing test does for that).